src/components/saved-cohorts/saved-cohorts.component.tsx

```diff
@@ -0,0 +1,185 @@
+import React, { useState } from "react";
+
+import { showToast } from "@openmrs/esm-framework";
+import {
+  DataTable,
+  Table,
+  TableHead,
+  TableRow,
+  TableHeader,
+  TableBody,
+  TableCell,
+  Pagination,
+  Search,
+  Button,
+  InlineLoading,
+} from "carbon-components-react";
+import { useTranslation } from "react-i18next";
+
+import mainStyles from "../../cohort-builder.scss";
+import { PaginationData } from "../../types";
+import EmptyData from "../empty-data/empty-data.component";
+import SavedCohortsOptions from "./saved-cohorts-options/saved-cohorts-options.component";
+import { deleteCohort, getCohorts } from "./saved-cohorts.resource";
+import styles from "./saved-cohorts.scss";
+
+interface SavedCohortsProps {
+  viewCohort: (queryId: string) => Promise<void>;
+}
+
+const SavedCohorts: React.FC<SavedCohortsProps> = ({ viewCohort }) => {
+  const [page, setPage] = useState(1);
```

Comment:
how about `currentPage` or `selectedPage`?

Reply:
this cannot be changed. It creates a conflict with the carbon pagination component pagination type